protobuf/OWSSignalServiceProtos.proto

```diff
@@ -124,12 +124,26 @@ message SyncMessage {
     optional uint64 timestamp = 2;
   }
 
-  optional Sent     sent     = 1;
-  optional Contacts contacts = 2;
-  optional Groups   groups   = 3;
-  optional Request  request  = 4;
-  repeated Read     read     = 5;
-  optional Blocked  blocked  = 6;
+  message Verification {
+    enum State {
+      DEFAULT            = 0;
+      VERIFIED           = 1;
+      NO_LONGER_VERIFIED = 2;
+    }
+
+    optional State  state       = 1;
+    // The e164 phone number of the user.
+    optional string destination = 2;
```

Comment:
Yup, I am deferring to the property names used by Moxie in his proto schema changes, since we're almost certainly going to adopt those once he's finalized them.